### PR TITLE
Add teuthology-update-inventory command

### PR DESCRIPTION
**Pull request by zmc**
This ought to also replace teuthology-updatekeys, but might not be as
fast yet.

Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

File: scripts/update_inventory.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+def main():
+    args = docopt.docopt(doc)
+    if args['--verbose']:
+        teuthology.log.setLevel(logging.DEBUG)
+
+    remotes = args.get('REMOTE')
+    for rem_name in remotes:
+        remote = teuthology.orchestra.remote.Remote(rem_name)
+        inventory_info = remote.inventory_info
+        del inventory_info['user']
```
Comment (andrewschoen) [inline scripts/update_inventory.py]: Why are we deleting this key? Maybe could use a comment to clarify.

File: scripts/update_inventory.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import docopt
+
+import teuthology
+import teuthology.config
+import teuthology.lock
+import teuthology.orchestra.remote
```
Comment (andrewschoen) [inline scripts/update_inventory.py]: Are all these imports needed? Looks like you're using the full path to the modules being used below.

File: scripts/update_inventory.py (hunk 1 above)
Comment (zmc) [inline scripts/update_inventory.py]: Ah, that turned out to be unnecessary. The user value would be incorrect, if paddles even used it. It doesn't yet, and when it does I'll have to fix it across the board. Removing that line.

File: scripts/update_inventory.py (hunk 2 above)
Comment (zmc) [inline scripts/update_inventory.py]: The .config import is unnecessary so I'm taking it out. The other two are needed, yes.

**Pull request merged**